README.md

```diff
@@ -1,101 +1,38 @@
-# HTML & CSS3 & JavaScript Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+## **TODO LIST**
+A simple app to help you manage the things you have to do in a day.<br />
 
-## Set-up GitHub Actions
+## Built With
 
-Please do the following **steps in this order**:
+- HTML,CSS, JavaScript
+- Frameworks, Webpack
+- Github, Linters
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+## Live Demo
 
-Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+[Live Demo Link](https://ice949.github.io/Webpack-Setup/)
```

Comment:
- [ ] [OPTIONAL] I noticed that your live demo is not working :cry: , I would like to recommend you fix it cause your readme file needed to be professional :+1:

Reply:
@codecaiine Alright Sir Noted with Thanks